great-idea/services.html

```diff
@@ -7,101 +7,83 @@
   <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/index2.css">
```

Comment:
Good job realising you wouyld need to change the link styled sheet if you wanted to work on a new CSS page. Steer away from naming it `index2.css` though. For organization and naming convention you could call it `services.css`